_sass/components/book.scss

```diff
@@ -79,6 +79,12 @@
 		margin-bottom: 2em;
 	}
 
+	ol ul,
+	ul ul {
```

Comment:
Although we don't have any yet, we should probably style for all four potential combinations:

```css
ul ul,
ul ol,
ol ol,
ol ul {
```